python/graphstorm/dataloading/sampler.py

```diff
@@ -389,3 +392,247 @@ def sample_blocks(self, g, seed_nodes, exclude_eids=None):
             blocks.insert(0, block)
 
         return seed_nodes, output_nodes, blocks
+
+class _FileSampler:
+    r"""File Sampler Interface.
+
+    Parameters:
+    ---------
+    dataset_path : str
+        Path to the data files.
+        Mutually exclusive with :attr:`files`.
+    """
+
+    def __init__(self, dataset_path=None):
+        self.dataset_path = None
+        self.files = None
+
+        if not dataset_path:
```

Comment:
If  dataset_path should not be None, why its default value is None. It should not have a default value.